hardware/plugins/Plugins.cpp

```diff
@@ -813,6 +834,7 @@ namespace Plugins {
 			if (!m_PyInterpreter)
 			{
 				_log.Log(LOG_ERROR, "(%s) failed to create interpreter.", m_PluginKey.c_str());
+				m_bIsStarting = false;
```

Comment:
Instead of 100 times writing m_bIsStarting=false and return false, i would (yes..) in this case use gotoExitFalse;
And add under in the function a
gotoExitFalse:
 m_bIsStarting=false;
 return false;

Reply:
OK, will fix.